Source/WebKit/UIProcess/API/C/curl/WKWebsiteDataStoreRefCurl.h

```diff
@@ -36,6 +36,9 @@ WK_EXPORT void WKWebsiteDataStoreEnableDefaultNetworkProxySettings(WKWebsiteData
 WK_EXPORT void WKWebsiteDataStoreEnableCustomNetworkProxySettings(WKWebsiteDataStoreRef, WKURLRef proxyUrl, WKStringRef ignoreHosts);
 WK_EXPORT void WKWebsiteDataStoreDisableNetworkProxySettings(WKWebsiteDataStoreRef);
 
+typedef void (*WKWebsiteDataStoreRemoveAlternativeServicesStorageFileCallback)(void* functionContext);
```

Comment:
It would be better to be heading towards an API design closer to WKWebsiteDataStore.allWebsiteDataTypes and removeDataOfTypes.